src/james.io.clss.pas

```diff
@@ -111,5 +118,23 @@ function TFile.Stream: IDataStream;
   end;
 end;
 
+function TFile.SysToUTF8(const s: String): String;
+begin
+  {$IFDEF FPC}
+    Result := LazUTF8.SysToUTF8(s)
```

Comment:
@nunopicado you can remove such functions `SysToUTF8` and `UTF8ToSys` and simplify your code.
These functions is not requered on this FPC version anymore.